frontend/packages/dev-console/integration-tests/features/Knative/sink-to-URI-ODC-3924.feature

```diff
@@ -0,0 +1,169 @@
+Feature: Event Sources can able sink to URI as well as Resource
+    User should be able to sink with resource as well as with URI
+
+
+Background: 
+    Given user has installed OpenShift Serverless Operator
+    And user is at Developer perspective
+
+
+@regression
+Scenario: Resource and URI radio button
```

Comment:
Update the scenario in more descriptive way, something like Event Sources page details

Reply:
Updated